voyager_archive/views_utils.py

```diff
@@ -1,14 +1,36 @@
 from django.shortcuts import get_object_or_404
+from pymarc import Record
 from .models import AuthRecord, BibRecord, MfhdRecord
 
 
 def get_auth_record(auth_id: int) -> AuthRecord:
-    return get_object_or_404(AuthRecord, auth_id=auth_id)
+    rec = AuthRecord.objects.get(auth_id=auth_id)
```

Comment:
These 2 lines can be combined: 
`rec = AuthRecord.objects.get(auth_id=auth_id).auth_record`